### PR TITLE
Basis for slash commands and /issue command

### PR DESCRIPTION
**Pull request by OxyOCE**
Still a WIP but figured this was the easiest way to get the base line in for slash commands as well as fast tracking a /issue command.

### CODE REVIEW

File: voluspa.py [hunk 1]
```diff
@@ -92,6 +93,8 @@ async def on_ready():
 
     bot.loop.create_task(update_status_task(bot, quotes))
 
+    await bot.tree.sync()
```
Comment (RecursiveHook) [inline voluspa.py:96]: What is this doing?

File: cogs/feedback.py [hunk 2]
```diff
@@ -1,39 +1,34 @@
 import logging
 
+from modules.ui_elements import FeedbackModal, IssueModal
+from modules.config import CONFIG
```
Comment (RecursiveHook) [inline cogs/feedback.py]: Am I missing it or is `modules.config` not included ATM??

**Comment by RecursiveHook** [comment]
Looks good so far, couple questions for you. :+1:

File: voluspa.py (hunk 1 above)
Comment (OxyOCE) [reply voluspa.py:96]: This syncs the application commands (slash commands) with discord at startup

[API Reference](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.CommandTree.sync)
[Example](https://github.com/Rapptz/discord.py/blob/master/examples/app_commands/basic.py#L28)

Comment (OxyOCE) [reply voluspa.py:96]: I.e. that's how you get the pretty discord context menu when you type /. Until now the paradigm has been Voluspa listens for commands in messages, now the command invocation is handled by Discord. That's my understanding.

**Review by RecursiveHook — CHANGES_REQUESTED** [review]

File: cogs/feedback.py (hunk 2 above)
Comment (OxyOCE) [reply cogs/feedback.py:4]: Ah - it seems I have changed it from `from voluspa import CONFIG` to this. Both work owing to the dict mashing magic that you did in modules/config.py, but if you want it changed back no worries. I don't thing there was a good reason I changed it.

Comment (OxyOCE) [reply cogs/feedback.py]: Oh I see what you mean - I offloaded the callback to ui_elements.py. I've removed it now.

Comment (RecursiveHook) [reply cogs/feedback.py]: Toight

**Pull request merged**